app/error.tsx

```diff
@@ -0,0 +1,34 @@
+'use client' // Error components must be Client Components
+ 
+import { Button } from '@/components/ui/button'
+import { useEffect } from 'react'
+ 
+export default function Error({
+  error,
+  reset,
+}: {
+  error: Error & { digest?: string }
+  reset: () => void
+}) {
+  useEffect(() => {
+    // TODO: Does vercel automatically log this?
+    console.error(error)
+  }, [error])
+  return (
+    <div className='flex justify-center items-center h-screen'>
+        <div className='prose pb-2'>
+            <div className='flex justify-center'>
+                <h1 className="mb-0">Oops!</h1>
+            </div>
+            <p>Sorry: The website encountered an error trying that operation! It returned this message:</p>
+            <div style={{ color: '#ff0000', backgroundColor: '#f5f5f5' }} className='rounded'>{error.message}</div>
+            <p>If you want that operation again, click below!</p>
+            <div className='w-full flex justify-center'>
+                <Button className="" onClick={() => reset()}>
```

Comment:
Can we add the tailwind class bg-background to this button --> will make the button the same color as the header (background is a cutom variable we have set in the config)

also in another siutation if we a prop is empty just remove it (classname is almost never required and if it is that should indicate an issue with the component structure)

Reply:
Done.